### ISSUE TITLE
Failing to launch sync or apply when disconnected from helm registry

### ISSUE THREAD

**Issue by sgandon** [roboll/helmfile#544]
When I am working from home (and do not run the vpn :) ) I cannot connect to our private helm registry.
But when I want to deploy my charts through one helmfile the process is stuck on one chart defined using the registry pattern.  The charts defined  using relative path are launched fine.
So it is expected ? 
is there a way to use some helm registry local cache ?

**Comment by mumoshu** [comment]
@sgandon Hey! Does `helmfile sync --skip-deps` work in that case?

**Comment by sgandon** [comment]
nope, it does not work either.
It seems to be stuck on the `helm upgrade`.

**Comment by mumoshu** [comment]
Thanks! Can we treat this as an upstream issue then?

**Comment by sgandon** [comment]
yes indeed, I'll create an issue in the helm project if not already done.

**Comment by sgandon** [comment]
https://github.com/helm/helm/issues/4618
